packages/dataviews/src/add-filter.js

```diff
@@ -172,7 +172,7 @@ export default function AddFilter( { filters, view, onChangeView } ) {
 												] ) => (
 													<DropdownMenuRadioItem
 														key={ key }
-														name={ `add-filter-${ filter.name }-conditions` }
+														name={ `add-filter-${ filter.field }-conditions` }
```

Comment:
This rendered `add-filter-Author-conditions` and now it renders `add-filter-author-conditions`. Note that fields could have names such as `Author Name`, so it's best to use the id.